1.10/installing/custom/system-requirements/install-docker-centos.md

```diff
@@ -20,6 +20,13 @@ In addition to the general [Docker requirements and recommendations for DC/OS][1
   mkfs -t xfs -n ftype=1 /dev/sdc1
   ```
 
+* If you are using Docker 1.12 or Docker 1.13, you must add `MountFlags=private` in `/etc/systemd/system/docker.service.d/execstart.conf` and then restart the Docker daemon with this command:
```

Comment:
I chatted with Docker folks. They haven't figured out the root cause yet. They did some work around. See https://github.com/moby/moby/issues/25718#issuecomment-313546920

It's indeed a Docker bug. I am not sure what should we do here.

One option I am leaning towards is just not mention it at all.